azurelinuxagent/common/errorstate.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime, timedelta
+
+ERROR_STATE_DELTA = timedelta(minutes=15)
+
+class ErrorState(object):
+    def __init__(self, min_timedelta):
```

Comment:
Might be cleaner to just manage the delta completely internally in this class, using the constant above.

Reply:
I like to be able to control this value for the purposes of testing. If I keep it in the class I cannot control it easily.